posts/progressive-enhancement/index.qmd

```diff
@@ -0,0 +1,152 @@
+---
+title: "Improving Ecosystem Interoperability Iteratively via Progressive Enhancement"
```

Comment:
I'm just realising we are inconsistent with the use of capitalisation and sentence case for our titles. We should probably harmonise this in a separate issue/PR.